testsuite/MDAnalysisTests/topology/test_tprparser.py

```diff
@@ -84,7 +85,7 @@ class TestTPRGromacsVersions(TPRAttrs):
     @pytest.fixture(params=[TPR400, TPR402, TPR403, TPR404, TPR405, TPR406,
                             TPR407, TPR450, TPR451, TPR452, TPR453, TPR454,
                             TPR455, TPR502, TPR504, TPR505, TPR510, TPR2016,
-                            TPR2018,])
+                            TPR2018, TPR2019B3])
```

Comment:
#effort

Reply:
?

Reply:
It's just funny how easy it is to add the new tests

Reply:
Yes! And it is especially nice since with nose I had to copy paste the whole class, which was super ugly!